README.md

```diff
@@ -23,13 +25,32 @@ A curated list of awesome projects built with [Hyperapp](https://github.com/hype
 - [hypernews](https://github.com/traducer/hypernews) - Hacker News clone with Hyperapp.
 
 ## Libraries and Mixins
+- [hyperapp-deepupdate](https://github.com/NoobLad/hyperapp-deepupdate) - A utility for updating deeply nested state.
+- [hyperapp-dot-notation-reducer](https://github.com/alber70g/hyperapp-dot-notation-reducer) - Allows actions to return an object with a path as a path as a property.
+- [hyperapp-firebase-auth](https://github.com/lukejacksonn/hyperapp-firebase-auth) - Drop in authentication for Hyperapp using Firebase.
+- [hyperapp-freeze](https://github.com/okwolf/hyperapp-freeze) - Deep freezes the state on load in development.
+- [hyperapp-hmr](https://github.com/scrapjs/hyperapp-hmr) - Hot module replacement for Hyperapp independent of the build process.
+- [hyperapp-html](https://github.com/swizz/hyperapp-html) - HTML helpers for Hyperapp.
+- [@hyperapp/logger](https://github.com/hyperapp/logger) - Logs state updates and action information to the console.
+- [hyperapp-partial](https://github.com/zaceno/hyperapp-partial) - Helps structure your code into related state properties, actions and events inside a namespace.
+- [hyperapp-persist](https://github.com/jamen/hyperapp-persist) - Store the state of Hyperapp in localStorage.
+- [hyperapp-redux-devtools](https://github.com/andyrj/hyperapp-redux-devtools) - Enables use of redux-devtools-extension for Hyperapp.
+- [@hyperapp/router](https://github.com/hyperapp/router) - The official router for Hyperapp.
+- [hyperapp-server](https://github.com/hyperapp/server) - Render Hyperapp statically from the server.
 - [hyperapp-transitions](https://github.com/zaceno/hyperapp-transitions) - Helps you animate transitions in your Hyperapp applications.
 
-## Resources and Others
+## Official Resources and Community
 - [Documentation](https://github.com/hyperapp/hyperapp/tree/master/docs) - Official Hyperapp documentation.
 - [Reddit](https://www.reddit.com/r/hyperapp/) - Official Hyperapp Subreddit.
 - [Slack](https://hyperappjs.herokuapp.com/) - Official Hyperapp community on Slack.
 - [Twitter](https://twitter.com/hyperappjs) - Official Hyperapp Twitter account.
+- [CodePen](https://codepen.io/hyperapp/) - Official Hyperapp Codepen account.
+
+## Other Resources
+- [How Hyperapp Works](https://gist.github.com/jbucaran/8dc33b7947f3193eb2ea3d5700e27036)
+- [Pure Programming with Hyper App](https://glebbahmutov.com/blog/pure-programming-with-hyper-app/)
```

Comment:
Hyperapp

Reply:
But it feels wrong to change the name of an article (which was exactly that), even if the title is wrong? Would it be better to omit it altogether?

Reply:
I think no one will think we are trying to be exact, but rather will end up looking sloppy. That's my opinion.